### ISSUE TITLE
Phar release is not created

### ISSUE THREAD

**Issue by asgrim** [Roave/BackwardCompatibilityCheck#374]
As per:

https://twitter.com/s_bergmann/status/1475014835129757699

Looks like the "release-phar" action did not run (can't verify, I'm on phone)

It should be triggered by creating the release. However, my guess is that because we're using laminas/automatic-releases, downstream events are not triggered due to the GitHub access token being used. It might be possible to switch this out to use the personal access token for the organisation, which would cause downstream events to trigger actions in turn.

https://docs.github.com/en/actions/reference/authentication-in-a-workflow#using-the-github_token-in-a-workflow

> When you use the repository's `GITHUB_TOKEN` to perform tasks on behalf of the GitHub Actions app, events triggered by the `GITHUB_TOKEN` will not create a new workflow run. This prevents you from accidentally creating recursive workflow runs. For example, if a workflow run pushes code using the repository's `GITHUB_TOKEN`, a new workflow will not run even when the repository contains a workflow configured to run when `push` events occur.

**Comment by Ocramius** [comment]
The token for the release is mine though: we should be fine.

We could copy https://github.com/maglnet/ComposerRequireChecker/blob/28d7833f000667fd90ae2d38b19cf670593af190/.github/workflows/phar-creation.yml to fix this.

**Comment by sebastianbergmann** [comment]
Sorry for bothering: **ping**.

**Comment by Legion112** [comment]
Should phar be attached release asset here?
When I will have time I might be willing to help to debug this one.
<img width="1397" alt="image" src="https://user-images.githubusercontent.com/20613430/205255488-1251a33a-eaeb-42d2-8316-0ebe2f20a5dd.png">

**Comment by sebastianbergmann** [comment]
> Should phar be attached release asset here?

Yes, please. This is where Psalm, for instance, publishes their PHARs: https://github.com/vimeo/psalm/releases/tag/5.6.0

**Comment by sebastianbergmann** [comment]
Any update on this? Thanks!

**Comment by Ocramius** [comment]
Nobody worked on Phar releases anymore, sorry: unlikely happening unless someone else proposes a patch for it.

**Comment by drupol** [comment]
Just made a proposal using Nix at https://github.com/Roave/BackwardCompatibilityCheck/pull/754

**Comment by sebastianbergmann** [comment]
This should be closed following #770.
